### PR TITLE
Update changelog for 0.5.12 release.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by bshastry — DISMISSED** [review]

**Comment by bshastry** [comment]
Needs rebasing

**Comment by chriseth** [comment]
Rebased.

File: Changelog.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Compiler Features:
 
 
 Bugfixes:
- * Fix internal error when popping a dynamic storage array of mappings.
- * Yul Optimizer: Fix reordering bug in connection with shifted one and mul/div-instructions in for loop conditions.
+ * Code Generator: Fix internal error when popping a dynamic storage array of mappings.
  * Scanner: Fix multi-line natspec comment parsing with triple slashes when file is encoded with CRLF instead of LF.
+ * Yul Optimizer: Fix reordering bug in connection with shifted one and mul/div-instructions in for loop conditions.
```
Comment (bshastry) [inline Changelog.md:19]: Could you please alphabetically sort Bugfix entries

**Review by bshastry — DISMISSED** [review]

**Comment by ekpyron** [comment]
If we're particularly picky, the last two "Compiler Features" entries need to be reversed as well...

**Comment by chriseth** [comment]
We are not.

**Pull request merged**